Documentation/saml-connector.md

```diff
@@ -0,0 +1,35 @@
+# Authentication through SAML
+
+## Overview
+
+This connector adds basic support for receiving authentication data through SAML. It uses a fork of `github.com/RobotsAndPencils/go-saml` hosted at `github.com/Calpicow/go-saml`.
```

Comment:
I've made a number of improvements to the existing code to support different SAML implementations and to perform sanity checks for security. All changes have a pending pull request. The maintainer is rather slow/reluctant to merge them, and I'd like to see progress now rather than later.